Sammuel09/myDiary
# Features

#159267412 Organize HTML files into UI folder

#### What does this PR do?
Organize HTML files into UI folder.

#### Description of the Task to be completed
Renders the frontend views of the myDairy app. 

#### How should this be manually tested?
After cloning the repo, open the UI folder in your text editor. Open the index.html in your browser to see the views.

#### What are the relevant pivotal tracker stories?
#159267412
